plugins/foreman_docker/1.x/index.md

```diff
@@ -0,0 +1,189 @@
+
+**Procedure 1.4. To Investigate a container:**
+
+1. Navigate to Containers → All Containers.
+2. On the Containers page, every Docker compute resource has a dedicated tab. Each of these tabs contains the table of available containers together with selected parameters of each container. Select the tab of the compute resource you want to inspect.
+3. To view the container metadata, click the name of the container you want to inspect. Foreman displays the table of container properties.
+4. On the Processes tab, you can view processes that are currently running in the container. Click on the process name to view the metadata of the process.
+5. If the container is running, you can view its standard output in the Logs tab. If you selected the allocate a pseudo-tty check box when creating a container, the console is interactive. Otherwise, it displays the initial standard output produced when the container started.
+
+## 1.1.3. Starting, committing, and removing containers
+A new container is by default disabled. By enabling a container, you start the processes of the containerized application in the compute resource. Hosts are then able to communicate with the container as with a web application. The following procedure shows how to start and stop a container:
+
+**Procedure 1.5. To start or stop a container:**
+
+1. Navigate to Containers → All Containers to view the list of available containers.
+2. Click Power On next to the container you want to start. After starting the container, the button changes to Power Off, which allows for stopping the container. These actions are equivalent to the docker start and docker stop commands.
+
+By committing a container, you create a new image layer that stores the status of the container. The following procedure shows how to commit a container:
+
+**Procedure 1.6. To commit a container:**
+
+1. Navigate to Containers → All Containers to view the list of available containers.
+2. Click the name of the container you want to commit.
+3. Click Commit. Foreman prompts you to provide the following information:
+    * Specify a repository name. This can be a single name or combined with the user name, for example user/my-rhel-image.
+    * Assign a tag to the image.
+    * Provide your contact information.
+    * Provide an informative comment about the image.
+4. Click Submit.
+
+The container is then committed to the repository of the original image. For example, if the container is based on an image pulled from the Docker Hub, the committed changes are pushed back to the Docker Hub.
+
+**Procedure 1.7. To remove a container:**
+
+1. Navigate to Containers → All Containers to view the list of available containers.
+2. Click the name of the container you want to delete.
+3. Click Delete.
+4. In the alert box, click OK to remove the container.
+⁠
+
+## 1.2. Working with repositories
+Follow the Katello [Docker documentation](http://www.katello.org/docs/2.2/user_guide/docker/docker.html).
```

Comment:
2.2?  Is there a better link?

Is there any non-Katello support that should be documented?